cogs/catgirl.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, bot):
         randCatgirl = random.choice(self.catgirls)
         embed = discord.Embed()
         embed.colour = discord.Colour.red()
-        embed.title = "Catgirl"
+        embed.title = "Catgirl"+randCatgirl[JSON_imageURLKey]
```

Comment:
These changes will only affect the title of the embed, not entirely sure about the context of this.

Reply:
Ah okay, I re-read `#feedback`, to me it seems like it's a Discord mobile thing.  Thoughts @KamiCreed ?  We can play with it on our staging bot and see how it is before it goes live.

Reply:
Actually, I'm not sure. Is this not the same as opening the image and opening in the browser?

Reply:
For mobile phones that is.

Reply:
The `Catgirl` title already links to the image URL, so adding that to the end doesn't make a difference.

Reply:
It seems to me that mobile saves it as the title. This is a quick and dirty fix; I could strip out the rest of the URL to get just the file extension at the end.

Reply:
Oh, I see. This is the title of the embed image, so the saved image from discord will take on this name. Though, if you put the URL there, I wonder what kind of name it will save? An invalid URL name?

Reply:
Probably should use python's path module thing to strip out the url and leave the filename and extension.

Reply:
Wow. I guess I didn't read your entire comment. Probably a good idea to do that.

Reply:
+1 on that.